src/Components/Attributes/Attributes.js

```diff
@@ -105,7 +109,17 @@ class Attributes extends React.Component {
     // `dataStructures` prop is optional and is used to
     // resolve inheritance, references and includes/mixins, plus to
     // render the inheritance tree.
-    const dataStructures = props.dataStructures || [];
+    let dataStructures = props.dataStructures || [];
+    let element = props.element;
+
+    if (isFunction(element.getMetaProperty)) {
+      // serialize minim Element inputs (`element` and `dataStructures`)
+      // to the expected refract 0.6 serialization
+      const refractSerializer = new JSON06Serialiser(minim.namespace());
```

Comment:
As you are only deserialising as apposed to serialising you don't actually need to provide a namespace to the serialiser, a default one will be created. This might help simplify the code.

Reply:
Yup, thanks.